linebot/exceptions.py

```diff
@@ -73,3 +73,12 @@ def __init__(self, status_code, error=None):
 
         self.status_code = status_code
         self.error = error
+
+    def __str__(self):
+        """str.
+
+        :rtype: str
+        :return:
+        """
+        return '{0}: error_response={1}'.format(
+            self.__class__.__name__, self.error)
```

Comment:
Showing also status code might help us debugging, how about?

Reply:
Ah, It is a good point !
I'll fix